python/adapters.h

```diff
@@ -1,15 +1,33 @@
 #pragma once
 
 #include <pybind11/pybind11.h>
+#include <pybind11/numpy.h>
 
 #include "utils.h"
 
 namespace py = pybind11;
 
-py::dict rosValueToDict(const Embag::RosValue::Pointer &ros_value);
-py::list rosValueToList(const Embag::RosValue::Pointer &ros_value);
+py::dict rosValueToDict(const Embag::RosValue::Pointer &ros_value, bool always_list_primitive_array);
+py::list rosValueToList(const Embag::RosValue::Pointer &ros_value, bool always_list_primitive_array);
```

Comment:
When would we want to force the list with `always_list_primitive_array`?

Reply:
The thinking here was that if you're trying to serialize the underlying data, you may just want to convert everything to a list. Doing the conversion in cpp land is faster _I think_, might want to double check now that I have the memoryview with the `to_list` functionality. One thing I just realized though - it might actually be a better interface to enable users to specify which `element_type` they want to have converted to a list vs not - this is something that would be super useful in some of our spark code for example where we want to leave uint8 types as a memoryview as we typically use them as raw bytes.

Reply:
Yeah that would actually be very helpful